build.xml

```diff
@@ -192,4 +190,21 @@
         <delete dir="${basedir}/${jre.filename.linux}/"/>
     </target>
     
+    <target name="dependency-report-html" depends="-load-ivy-tasks,-init-build-dir" description="Generate a detailed dependency report">
+        <delete dir="${suite.build.dir}/ivy-reports"/>
+        <mkdir dir="${suite.build.dir}/ivy-reports" />
+        
+        <ivy:resolve type="${ivy.resolve.types}"/>
+        <ivy:report todir="${suite.build.dir}/ivy-reports" graph="false" xml="false"/>
+    </target>
+    
+    <target name="check-latest-version" depends="-load-ivy-tasks" description="Check whether there is a new release of a dependency.">
+        <ivy:resolve/>
+        <ivy:checkdepsupdate showTransitive="false" revisionToCheck="latest.release"/>
+    </target>
+    
+    <target name="dependency-tree" depends="-load-ivy-tasks" description="Output the dependencies ina tree layout to the console.">
```

Comment:
typo "ina" change to "in a"

Reply:
Thank you @rsabhi. I've corrected it, please check.